### ISSUE TITLE
Accessibility Bug: When the theme is switched to High Contrast Black theme, the icons are not visible in the screen.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3151]
Fundamental core component: "Icon tab"

Description: When the theme is switched to High Contrast Black theme, the icons are not visible in the screen.

Expected: All the UI elements should get adapt to the High contrast Black theme when the theme is applied.

Attached screenshot:

In Fiori 3 theme

![image](https://user-images.githubusercontent.com/32538291/91651684-5edcec80-eaad-11ea-897f-16cad7d97724.png)

In High Contrast Black theme

![image](https://user-images.githubusercontent.com/32538291/91651692-6ac8ae80-eaad-11ea-9bc8-3f0afadffe58.png)

**Comment by puru-hk** [comment]
This is observed in all the components.

**Comment by droshev** [comment]
I cannot reproduce the exact issue
![Screen Shot 2020-09-17 at 5 40 12 PM](https://user-images.githubusercontent.com/4380815/93531241-e24f6800-f90c-11ea-9ebf-157bb851a3c4.png)
The issue on my end is that the names are in black
